15.-Project-3-Elections-Scraper.py

```diff
@@ -0,0 +1,155 @@
+
+import requests
+from bs4 import BeautifulSoup as BS
+import csv
+import sys
+import os
+import unicodedata
+
+URL = 'https://volby.cz/pls/ps2017nss/ps3?xjazyk=CZ'
+payload = {'area':None,
+           'municipality': None}
+
+
+def parse_command_line():
+    while True:
+        if len(sys.argv) != 3:
+            print('Not correct number of arguments (2)')
+            break
+        else:
+            arg = sys.argv[1:]
+            count = 0
+            for k, v in payload.items():
+                payload[k] = arg[count]
+                count += 1
+            return payload
+
+
+def scrape_area():
+    parsed_command_line = parse_command_line()
```

Comment:
You have already command-line arguments parsed. You should pass the dictionary not do the work again